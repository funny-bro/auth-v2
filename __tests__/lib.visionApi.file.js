const {fileToBase64} = require('../lib/visionApi/file')

describe('', ()=>{
  it('', ()=>{
    const base64 = fileToBase64(`${__dirname}/../mock/b235r.jpg`)
    expect(base64).toMatchSnapshot();
  })
})