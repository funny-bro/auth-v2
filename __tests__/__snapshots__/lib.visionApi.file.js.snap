// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `"/9j/4AAQSkZJRgABAQEASABIAAD/2wBDAAMCAgMCAgMDAwMEAwMEBQgFBQQEBQoHBwYIDAoMDAsKCwsNDhIQDQ4RDgsLEBYQERMUFRUVDA8XGBYUGBIUFRT/2wBDAQMEBAUEBQkFBQkUDQsNFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBT/wAARCAAyAMgDASIAAhEBAxEB/8QAHwAAAQUBAQEBAQEAAAAAAAAAAAECAwQFBgcICQoL/8QAtRAAAgEDAwIEAwUFBAQAAAF9AQIDAAQRBRIhMUEGE1FhByJxFDKBkaEII0KxwRVS0fAkM2JyggkKFhcYGRolJicoKSo0NTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uHi4+Tl5ufo6erx8vP09fb3+Pn6/8QAHwEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoL/8QAtREAAgECBAQDBAcFBAQAAQJ3AAECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRChYkNOEl8RcYGRomJygpKjU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6goOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4uPk5ebn6Onq8vP09fb3+Pn6/9oADAMBAAIRAxEAPwD5ct1o1jxHpvhWyju9Uufstu8giV/LZ8sQTjCgnoDUtuvSuE+PYx4Lsv8Ar/T/ANFyUAek+FfEmm+K7J7vSrn7VbpIYmfy2TDAA4wwB6MPzrp7delePfs6OkXge/kkZURdQkLMxwAPKi5Jr1PSfEuj6pceRZ6tY3c46xQXKO35A5oA6G3XpWnbrXn3jv4p6P8ADqOCO8Et3qE4zDY2wBkYZxuOegzx79gcGudtf2jZNLuYD4j8GaroOnzMFS8kDMOe+GRcjvwSfY0Ae4tcwWMDT3M0dvCnLSSuFUfUngVDpnj3wvfXK21t4k0i4uCcCGK+iZyfoGzXmnj34MRfGTxBpurSa/JD4fWxjEUFr8/msWdjICTtGVZOcEnHsK81+NH7NGl+AfBNx4h0XU72ZbRoxPBfFG3K7hAVKquCCy8EHjNAH2Pbr0rTt16V8gfAX426p4V+Dvi2/wBQgk1uDw5JafZYpZ9hKTyeX5e/a2AuMjg9ccDGO50/9p/xnfWkF1afBjXru2nRZYpoZJnSRCMqysLbBBBBBFAH0zbr0rUt16V4x8NvjjPrtpqd3428NyfDS1tXhit59fuTCl28gkJVGljjGVEecDPXtivVdA8WaFr+mTajpetadqWnQMUlu7S7jlijYAEhnUkAgEHBPQj1oA6W3XpWnbrXP+GvEekeKLaS50bVbLVreKUwyS2NwkyJIACUJUkBgGBx15HrXSW69KAL9uvStO3XpVC3XpWpbr0oAvW61qW69KoW69K07daAL9uvStO3WqNuvStO3XpQBet16Vp269Ko269K07delAF63XpWpbr0qhbr0rTt16UAX7delFS269KKAPw3t16VwXx+GPBVl/2EE/8ARclehW69K4D9oEY8E2X/AGEE/wDRclAHCfDnwfr3xA0GTSLa5Gn6FFdNPcXDAnzJCqALtBG7AUHGQOfpVX4jfC3UPhhLZXaXwu7aZ8RXUSmJ0kHOCMnHqCD2PSvUv2aJo28H6hCJFMy37O0efmCmOMA49CQfyNM/ad1a3j8N6Tpe8G7kuvtITPIRUdcn6lx+R9KAN74TaXZfExNB8banLPLrulI1gyhl8l2XeQ7LtzuxKG4I5xxXcfGe802y+FviD+1ChjltmigV+rTkfu8D1DYP4E9q85+CGu2Pw4+Dc2ta1N5Ftc3ss0KY+eX5VQKg7kmM/wAzxzWdpOkal8bdSl8ZeMS+l+BtLR54bMbiJI0BLYAGSML8zgZOML7AHV+H/iPdfCD9nzwjqt3pjapLPL9nWFp/JKo5mkjbO1v4FXjHfrXlHxH/AGg9S+K8Fvot3HF4e0B5lefyAbiRsHgsfl3AdcADn8K+u/BvijRPFOl2tzol5DcWjx7olRTGwjV2jzsYBgu5GUcAfKcVzPx98HeG9V+Gev6hqtraw3lpavNbXpQLKJgPkUN1O5sLjvmgDz/xD4f8JaD+yT4iHhHUF1a2me2e5viMSSTfaYch16pgYwp6D1zk3PAn7TesaF4C0KCz+GetarpOmafBazarEziImONUZgREy4+Xu1cV+xZG2q+K/E2jXdul5od1pu+5tp0DxO6yoE3KeDw0lfcFjbx28UcUUaxRIoVEQYVQOAAOwoA+Sf2lPi5oPxg+AGl6noskiPDr8Ed1Z3AAlgc29xgMASCDg4I4OD3BA8++A/wg8a/HLwvc+H7HVRovgq0v2urmeRSyy3LJGu0ICPMZVRDgkBd2c5YZwP2irS20f4reNbDRVVNE/tC3kkSL/VrceSxYYHAw7zjHbkV9Vf8ABP0Z+D2sf9h6b/0nt6APVv2dvgp/worwbeaD/bP9ufadQe++0fZfs+3dHGmzbvfP+rznPfpxXsFutUbdeladuvSgC9br0rUt16VQt1rUt16UAXrda07delUbdeladuvSgC/br0rSt16VRt16Vp260AXrdelaluvSqFuvStO3XpQBft16Vp269Ko269K07delAF63XpRUtuvSigD8N7delc/8T/BN9468O21hYS28M0d0s5a5ZlXaEdewPOWFdLbr0rUt16UAePaX+z3qNho0E1nrrab4iRm3zW0jiJlJ+VQwAYcDOcd+nel0f9mvWNX1gXfijXo7mMsDIYJHmmlA7b3Ax9ea9zt16Vp269KAPFfiP8BvEfi3XrZ9HvdNttEsoI4bKyupJAIQoGflEbDkjvnPetG4+HXxn1HR59Kl8V6D/Z88DWr26QqimJl2lRtthgYOOMY7V7Zbr0rTt16UAeAn9lnVrrwz4e8rxGmmeItKt5LYvbb2hdTcSyqVcbXU/vcE4PTp3Ofdfsr/ABH8VTRReIfGtve2iHKtPeXN0ye4R1A6Z719Q269K1LdelAHjUH7LNppXgJND8PeIrzRtVN0l7PrEafvJ5EVgi4VlKoNxIAPBOTmq6fCv4/iA6dF8S9NOnEbDOyEXGPXd5BbP/A/xr6At16Vp269KAPmjxN+xpcyfCpNB0HVbW88SXWrRalqOq6szxrMFimXau1XIwZc85zliT0Fetfss/B7Wfgr4Cv9E1y5sbq7uNTkvUfT5HeMI0USAEuinOY27Y5HNeoW69K1LdelAF63WtO3XpVC3XpWnbr0oAv269K07dao269K07delAF+3XpWnbr0qhbrWpbr0oAvW61p269Ko269K07delAF63XpWnbr0qjbr0rTt16UAX7deladuvSqFuvStS3XpQBet16UVLbr0ooA/Da37Vp2/SiigDTt+tadv2oooA1LetK37UUUAalv2rTt+tFFAGnb9q07eiigDUt+1adv2oooA0rfpWpb9aKKANO37Vp29FFAGpbdq07btRRQBp2/WtO37UUUAadvWnb9qKKANS27Vp29FFAGnb0UUUAf/9k="`;
